src/V3Assert.cpp

```diff
@@ -518,6 +518,54 @@ class AssertVisitor final : public VNVisitor {
         iterateChildren(nodep);
         newPslAssertion(nodep, nodep->failsp());
     }
+    void visit(AstAssertCtl* nodep) override {
+        if (VN_IS(m_modp, Class) || VN_IS(m_modp, Iface)) {
+            nodep->v3warn(E_UNSUPPORTED, "Unsupported: assertcontrols in classes or interfaces");
+            pushDeletep(nodep->unlinkFrBack());
```

Comment:
```suggestion
            VL_DO_DANGLING(pushDeletep(nodep->unlinkFrBack()), nodep);
```

Sorry missed this last time!